src/Microsoft.PowerShell.Commands.Management/commands/management/Navigation.cs

```diff
@@ -3266,6 +3266,10 @@ protected override void ProcessRecord()
                     {
                         // not a directory
                     }
+                    catch (System.ArgumentException)
+                    {
```

Comment:
Need to make sure that FileSystem paths continue to work as they were before this new catch handler; 2 tests:
1) remove-item with -Path and filesystem path that contains *
2) remove-item with -LiteralPath and filesystem path that contains *

Reply:
Looks like there's tests for -Path, I'll add a test for -LiteralPath with *